14-operations/manifest.yaml

```diff
@@ -0,0 +1,96 @@
+version: "2"
+name: "Operations"
+description: "Pack of operations"
+
+service_account:
+  display_name: Operations Bot
+
+functions:
+  - name: operation_handler
+    description: function to handle operations
+
+operations:
+  - name: get_temperature
+    display_name: Get Temperature
+    description: Operation to get the temperature of a city
+    slug: get_temperature
+    function: operation_handler
+    type: action
+    inputs:
```

Comment:
Maybe we can add some comments explaining what these inputs are and how they are different from snap-in inputs.